docs/_data/meetups/ger-munich.yaml

```diff
@@ -5,8 +5,8 @@ website: https://www.meetup.com/write-the-docs-muc/
 meetup: write-the-docs-muc
 organizers:
   - name: Ekaterina Ivanova ('@i.kate')
-    link: https://writethedocs.slack.com/messages/@UCRH2R4HL
+    link: https://www.meetup.com/write-the-docs-muc/members/183783654/
```

Comment:
haha, cannot believe I didn't see this before!

Reply:
Many eyes :)